### PR TITLE
Merge clonality branch

### PR DESCRIPTION
**Pull request by leeprichman**
* added RNA filter to garnish_predictions
* added clonality into model and create sample level garnish_score
* means cluster cell fractions to rank clones
* added garnish_targets

### CODE REVIEW

File: R/antigen.garnish_fitness.R [hunk 1]
```diff
@@ -335,3 +335,108 @@ lapply(dtl %>% seq_along, function(i){
...
 
 }
+
+
+## ---- garnish_clonality
+#' Internal function to integrate clonality data into final `garnish_score` parameter.
+#'
+#' Integrates clonality input for creation of a summary metric of tumor fitness similar to [Luksza et al. *Nature* 2017](https://www.ncbi.nlm.nih.gov/pubmed/29132144).
+#'
+#' @param dt Data table. ##### TODO
```
Comment (andrewrech) [inline R/antigen.garnish_fitness.R]: #91 @leeprichman

**Comment by andrewrech** [comment]
TODO left for me on this merge

* [ ] code deep dive
* [ ] run large human data set
* [ ]  proof read docs again

@leeprichman

**Comment by leeprichman** [comment]
I will pull latest update docs that we talked about, change how variants handles prop_tab, and fix conflicts and if CI is good I will merge

**Comment by leeprichman** [comment]
final TODO

[ ] add ECDF functionality to garnish_clonality then update tests
[ ] merge

**Comment by leeprichman** [comment]
@andrewrech ready to merge once you take a last look.

**Comment by andrewrech** [comment]
@davidballi this branch is going to be merged in soon and has significant changes as noted above, mostly related to clonality and simplifying `garnish_variants`

**Comment by davidballi** [comment]
cool.  I will pull after merge is complete.  Still have need for a `garnish_slim()` function?

**Comment by andrewrech** [comment]
yea, if anything, more need now

**Pull request merged**